src/fibonacci.js

```diff
@@ -0,0 +1,13 @@
+export default function fibonacci(num){
+	if (num === 1) {
+		return [0]
+	} else if (num > 1) {
```

Comment:
you should account for the num 2 case because in the fibonacci algorithm that also returns a 1